sdk/test-utils/perfstress/test/sleep.spec.ts

```diff
@@ -5,7 +5,7 @@ import { PerfStressTest } from "../src";
 import { delay } from "@azure/core-http";
 
 // Used for verifying the perf framework correctly computes average throughput across parallel tests of different speed
-export class SleepTest extends PerfStressTest<string> {
+export class SleepTest extends PerfStressTest<{}> {
```

Comment:
So! I like all of these `<{}>`, they look super cool to me. However, is this the only way? Would it be possible to have a default type and skip the `<{}>` entirely?

Reply:
You said.. I did.